src/People.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      people: [],
+      info: {},
+      error: false,
+      input: "",
+      age: "",
```

Comment:
You can avoid having so many states by just putting the person in the state and then you can access age, name, and gender right from there